catalyst_voices/apps/voices/lib/pages/proposal/proposal_back.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:catalyst_voices/widgets/widgets.dart';
+import 'package:catalyst_voices_localization/catalyst_voices_localization.dart';
+import 'package:flutter/material.dart';
+
+class ProposalBack extends StatelessWidget {
```

Comment:
Current name feels like it is missing "something", could we rename it to:

```suggestion
class ProposalBackButton extends StatelessWidget {
```